test/blackbox-tests/test-cases/odoc-package-mld-link/run.t

```diff
@@ -0,0 +1,10 @@
+Make sure that links between mld files are resolved even when there is
+no library associated with the project
+
+This test case is based on code provided by @vphantom, ocaml/dune#2007
+
+  $ dune build @doc
+
+  $ grep -r xref-unresolved _build/default/_doc/_html/odoc_page_link_bug/index.html
```

Comment:
It might make sense of just build this target directly. It will make the tests a tad quicker.

Reply:
Ah, good idea, fixed.